lib/dal/src/func/view.rs

```diff
@@ -0,0 +1,4 @@
+mod list;
+
+pub use list::FuncSummary;
+pub use list::FuncSummaryError;
```

Comment:
This is a relatively empty module, but we know we will need it for the "get func" stuff soon